xfrd-tcp.c

```diff
@@ -737,7 +988,33 @@ xfrd_tcp_write(struct xfrd_tcp_pipeline* tp, xfrd_zone_type* zone)
 			return;
 		}
 	}
-	ret = conn_write(tcp);
+#ifdef HAVE_TLS_1_3
+	if (tp->ssl) {
+		if(tp->handshake_done) {
+			ret = conn_write_ssl(tcp, tp->ssl);
+
+		} else if(ssl_handshake(tp)) {
+			tcp_pipe_reset_timeout(tp); /* reschedule */
+			return;
+
+		} else {
+			if(tp->handshake_want == SSL_ERROR_SYSCALL) {
+				log_msg(LOG_ERR, "xfrd: TLS handshake failed: %s",
+					strerror(errno));
+
+			} else if(tp->handshake_want == SSL_ERROR_SSL) {
+				if (SSL_get_verify_result(tp->ssl) == X509_V_OK)
+					log_crypto_err("xfrd: TLS handshake failed");
```

Comment:
```suggestion
			} else if(tp->handshake_want == SSL_ERROR_SSL) {
				log_crypto_err("xfrd: TLS handshake failed");
```
Same as the other get_verify_result comment?

Reply:
Yes I'll remove that extra test...